woocommerce/woocommerce
# Create Add to Cart With Options wrapper block

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR creates an `add-to-cart-with-options` wrapper block.

![Screenshot 2024-11-26 at 7 33 45 PM](https://github.com/user-attachments/assets/372854fb-59b1-4bf3-8922-d3635f83a24a)

Closes #53008.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install and activate the `WooCommerce Beta Tester` plugin
2. Go to `WooCommerce Admin Test Helper` -> `Features` and enable `blockified-add-to-cart`.
3. Create a new product
4. Go to Pages > Add New Page
5. Add a `Single product` block and select the product you just created.
6. Go to the block inserter and add an `Add to Cart with Options (Experimental)` block.
7. It should look and behave exactly like the `Add to Cart with Options` block.



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
